selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceProjects.java

```diff
@@ -11,39 +11,148 @@
+import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceProjects.Locators.DELETE_BUTTON_XPATH;
+import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceProjects.Locators.NOTIFICATION_CONTAINER_ID;
+import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceProjects.Locators.SEARCH_FIELD_XPATH;
+import static org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceProjects.Locators.SELECT_ALL_CHECKBOX_XPATH;
 import static org.openqa.selenium.support.ui.ExpectedConditions.invisibilityOfElementLocated;
 import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
 import static org.testng.Assert.fail;
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.eclipse.che.selenium.core.webdriver.SeleniumWebDriverHelper;
 import org.openqa.selenium.By;
 import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-/** @author Musienko Maxim */
+/**
+ * @author Musienko Maxim
+ * @author Ihor Okhrimenko
+ */
 @Singleton
 public class WorkspaceProjects {
 
   private final SeleniumWebDriver seleniumWebDriver;
+  private final SeleniumWebDriverHelper seleniumWebDriverHelper;
 
   @Inject
-  public WorkspaceProjects(SeleniumWebDriver seleniumWebDriver) {
+  public WorkspaceProjects(
+      SeleniumWebDriver seleniumWebDriver, SeleniumWebDriverHelper seleniumWebDriverHelper) {
     this.seleniumWebDriver = seleniumWebDriver;
+    this.seleniumWebDriverHelper = seleniumWebDriverHelper;
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
-  private interface Locators {
+  public interface Locators {
     String PROJECT_BY_NAME = "//div[contains(@ng-click, 'projectItem')]/span[text()='%s']";
     String DELETE_PROJECT = "//button/span[text()='Delete']";
     String DELETE_SELECTED_PROJECTS = "//che-button-primary[@che-button-title='Delete']/button";
     String DELETE_IT_PROJECT = "//md-dialog//che-button-primary[@che-button-title='Delete']/button";
     String ADD_NEW_PROJECT_BUTTON = "//che-button-primary[@che-button-title='Add Project']/button";
     String PROJECT_CHECKBOX = "//md-checkbox[contains(@aria-label, 'Project %s')]";
+    String SEARCH_FIELD_XPATH = "//workspace-details-projects//input[@placeholder='Search']";
+    String NOTIFICATION_CONTAINER_ID = "che-notification-container";
+    String DELETE_BUTTON_XPATH = "//che-button-primary[@che-button-title='Delete']";
+    String DELETE_BUTTON_MESSAGE_XPATH = "//che-list-header-delete-button/span";
+    String SELECT_ALL_CHECKBOX_XPATH = "//md-checkbox[@aria-label='Project list']/div";
+    String CHECKBOX_BY_PROJECT_NAME_XPATH_TEMPLATE = "//md-checkbox[@aria-label='Project %s']";
+  }
+
+  public enum BottomButton {
```

Comment:
Position of buttons could be changed later, so it would be better not to use position in enum name to avoid misleading. What about `ActionButton` name instead of `BottomButton`?